test.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+
+
+<body>
+<div class="container">

```

Comment:
Gutter can be implemented by both padding and margin.
If using padding, all columns will have left and right paddings and container will have -ve padding to compensate first and last elements padding
If margin is used then size of each column will be 
     ( $container_width - ($total_columns -1) * $gutter ) / $total_columns